identity-wallet/src/state/reducers/history.rs

```diff
@@ -0,0 +1,29 @@
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, TS)]
+#[ts(export)]
+pub enum EventType {
+    ConnectionAdded,
+    CredentialsAdded,
+    CredentialsShared,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, TS)]
+#[ts(export)]
+pub struct HistoryEvent {
+    pub issuer_name: String,
```

Comment:
Probably better to change this to a more generic name, such as `name` or `connection_name`. 

In most cases issuers and verifiers are very distinct entities so `issuer_name` is not the right term when credentials are shared.